lib/commons/dom/inserted-into-focus-order.js

```diff
@@ -12,13 +12,7 @@ import isNativelyFocusable from './is-natively-focusable';
  * if its tabindex were removed. Else, false.
  */
 function insertedIntoFocusOrder(el) {
-  const tabIndex = parseInt(el.getAttribute('tabindex'), 10);
-
-  // an element that has an invalid tabindex will return 0 or -1 based on
-  // if it is natively focusable or not, which will always be false for this
-  // check as NaN is not > 1
-  // @see https://www.w3.org/TR/html51/editing.html#the-tabindex-attribute
-  return tabIndex > -1 && isFocusable(el) && !isNativelyFocusable(el);
+  return isInTabOrder(el) && !isNativelyFocusable(el);
```

Comment:
While this is functionally the same, by doing it this way you're running a bunch of unnecessary code in isFocusable. Best to revert here too.